src/scripts/modules/media/footer/footer.less

```diff
@@ -39,9 +39,6 @@
       }
 
       &:focus {
-        border-top: 0.1rem solid @brand-primary;
-        border-left: 0.1rem solid @brand-primary;
-        border-right: 0.1rem solid @brand-primary;
         outline: 0;
```

Comment:
[accessibility]
When you use `tab` key to navigate through the website, you should be able to see focus on elements. 
Those rules which you have removed actually disable the default focus visibility and add borders instead. 

My suggestion is to remove the 41 and 42 lines too and there will be default focus which shouldn't affect the layout.

Reply:
I've changed to:
```
outline: 1px;
outline-style: solid;
```
This is giving same result as setting `border` but will not cause moving elements.